src/RacingCarGame.js

```diff
@@ -0,0 +1,101 @@
+// eslint-disable-next-line import/extensions
+import Car from './Car.js';
+
+function isValidCarNames(carNames) {
+  if (carNames === '') {
+    return false;
+  }
+  const arr = carNames.split(',');
+  if (!arr.every((element) => element.length <= 5)) {
+    return false;
+  }
+  const set = new Set(arr);
+  return arr.length === set.size;
+}
+
+export default class RacingCarGame {
+  constructor() {
+    this.$carGameContainer = document.querySelector('.car-game-container');
+    this.$carNamesInput = document.querySelector('#car-names-input');
+    this.$racingCountInput = document.querySelector('#racing-count-input');
+    this.$result = document.querySelectorAll('div');
+    this.cars = [];
+  }
+
+  addEventListener() {
+    this.$carGameContainer.addEventListener('click', (e) => {
+      if (e.target.id === 'car-names-submit') {
+        this.onClickCarNamesSubmit();
+      } else if (e.target.id === 'racing-count-submit') {
+        this.onClickRacingCountSubmit();
+      }
+    });
+  }
+
+  onClickCarNamesSubmit() {
+    this.cars.length = 0;
+    const carNames = this.$carNamesInput.value;
+    if (isValidCarNames(carNames)) {
+      const carNamesArr = carNames.split(',');
```

Comment:
```suggestion
      const carNamesArr = carNames.split(',').trim();
```

trim으로 공백처리를 하면 더 좋을 듯 합니다.